qmlbind/src/api_application.cpp

```diff
@@ -91,6 +91,11 @@ qmlbind_application qmlbind_application_new(int argc, char **argv)
     return app;
 }
 
+qmlbind_application qmlbind_application_instance(void)
+{
+    return static_cast <QApplication*> (QCoreApplication::instance());
```

Comment:
How about using `http://doc.qt.io/qt-5/qapplication.html#qApp` macro?
It doesn't need static cast.